04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb

```diff
@@ -295,7 +295,9 @@
     "<a class=\"anchor\" id=\"Master_Flat\"></a>\n",
     "# Create master flat field\n",
     "\n",
-    "The GSAOI Kshort master flat is created from a series of lamp-on and lamp-off dome exposures. They should all have the same exposure time. Each flavor is stacked (averaged), then the lamp-off stack is subtracted from the lamp-on stack, and the result is normalized."
+    "The GSAOI Kshort master flat is created from a series of lamp-on and lamp-off dome exposures. They should all have the same exposure time. Each flavor is stacked (averaged), then the lamp-off stack is subtracted from the lamp-on stack, and the result is normalized.\n",
+    "\n",
+    "The master flat will be saved as S20170505S0030_flat.fits."
```

Comment:
Change
_"The master flat will be saved as S20170505S0030\_flat.fits."_
with
_"The master flat will be saved with the \_flat.fits."_